presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeExecution.java

```diff
@@ -87,4 +93,17 @@ public void testNativeExecutionShuffleManager()
         assertEquals(0, shuffleId);
         assertEquals(shuffleManager.getNumOfPartitions(shuffleId), bypassMergeSortShuffleHandle.numMaps());
     }
+
+    /**
+     * Test native execution of a cpp function declared via json file.
+     */
+    @Test
+    public void testNativeExecutionWithJsonFileBasedFunction()
```

Comment:
All tests in this file are for native execution, hence, including "NativeExecution" in the test name is redundant. Consider renaming to "testJsonFileBasedFunction"

Reply:
Good call, was a "consistent" CP from last version of this file but things I changed. Let me catch this up.